### DISCUSSION TITLE
Which type to choose for 'application/json': File or DigitalDocument

### DISCUSSION THREAD

**Discussion by SteffenBrinckmann** [TheELNConsortium/TheELNFileFormat#4] [General]
Hey,
so json files are just files; that is logical. But they are more as they contain metadata information from the database. Could we agree on giving them the type "DigitalDocument" which according to schema.org is "An electronic file or document"? If we call those entries Digital Document, and the Datafiles as "Files" then the import parser should be easier for all.

**Comment by NicolasCARPi** [comment]
Wouldn't it make more sense to integrate them directly in the metadata.json somehow and leave the `Files` for Datafiles only?

**Comment by SteffenBrinckmann** [comment]
The issue is that the json files can be very big as i store thumbnails and
lots of metadata in then. Parsing large files can be an issue.
I could leave the thumbnails and metadata out, but that would not be the
point.

**Reply by NicolasCARPi** [reply]
So you base64 encode the thumbnails into json? Why not saving them as `File` instead?

**Comment by SteffenBrinckmann** [comment]
There are possibilities to make the for export into eln more and more
intricate/ complicated. As always, i prefer to get an easy / lazy version
working, and if that does not do the trick, i can make it more complicated.
If it does the trick, simple it stays.

I might iterate it, depending what i see from others.

Keep you posted
